jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/deadlines/notifications/impl/email/EmailNotificationListener.java

```diff
@@ -60,19 +60,14 @@
     
     private static final Logger logger = LoggerFactory.getLogger(EmailNotificationListener.class);
 
-    private UserInfo userInfo;
-
     private Session mailSession = EmailSessionProducer.produceSession();
     
-    public EmailNotificationListener(UserInfo userInfo) {
-    	this.userInfo = userInfo;
-    }
     
     @Override
-    public void onNotification(NotificationEvent event) {
-
+    public void onNotification(NotificationEvent event, UserInfo userInfo) {
+        
         if (userInfo == null || mailSession == null) {
```

Comment:
it still can be null as it would then mean this (EmailNotificationListener) listener is disabled